Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -78,19 +78,14 @@ private void ResetConfiguration()
 
         #region Active SDK components
 
-        /// <summary>
-        /// The Active Controllers property lists all the controllers detected by the Mixed Reality manager on startup
-        /// </summary>
-        //[SerializeField]
-        //[Tooltip("The collection of currently active / detected controllers")]
-        //private Controller[] activeControllers = null;
-
         ///// <summary>
         ///// The Active Headset property maintains the Headsets/SDK detected by the Mixed Reality manager on startup
         ///// </summary>
-        //[SerializeField]
-        //[Tooltip("The currently active / detected Headset or SDK")]
-        //private Headset activeHeadset = default(Headset);
+        [SerializeField]
+        [Tooltip("The currently active / detected Headset or SDK")]
+        private IMixedRealityDevice activeDevice = default(IMixedRealityDevice);
+
+        public IMixedRealityDevice ActiveDevice { get { return activeDevice; } }
```

Comment:
I like this a bit better, but wondering if it'll limit us to how we use the controllers.

Reply:
We can review as we progress, as it's the Interface it implements, we have a bit more freedom. If it was a concrete class we would have issue.